packages/eslint-config-base/rules/filenames.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  plugins: ['filenames'],
+  overrides: [
```

Comment:
Does this maybe make sense to add the the jest config instead? Since it specific to jest tests.
We will probably also want similar for the cypress extension (*.cy.*) in the cypress config

Reply:
Since the proposal advocates for specific naming regardless of which unit test runner we have, I thought it might make more sense to keep that rule in our base config since it will be true regardless of a project using Jest or not. Thoughts?

Reply:
I went ahead and added this to the Jest config.